clang/lib/Format/Format.cpp

```diff
@@ -1501,6 +1536,8 @@ FormatStyle getLLVMStyle(FormatStyle::LanguageKind Language) {
   LLVMStyle.AllowShortCompoundRequirementOnASingleLine = true;
   LLVMStyle.AllowShortEnumsOnASingleLine = true;
   LLVMStyle.AllowShortFunctionsOnASingleLine = FormatStyle::SFS_All;
+  LLVMStyle.AllowShortFunctionsOnASingleLineOptions = {};
```

Comment:
```suggestion
  LLVMStyle.AllowShortFunctionsOnASingleLineOptions.Empty = true;
  LLVMStyle.AllowShortFunctionsOnASingleLineOptions.Inline = true;
```
`SFS_All` requires setting everything.

Reply:
Thanks, fixed!